app/assets/stylesheets/application.scss

```diff
@@ -196,3 +201,9 @@ body {
     }
   }
 }
+
+.inherit-link-color {
```

Comment:
based on the name of this class I would expect it to be a class applied to the link itself (instead of its parent). would that approach work here? I think it's also a bit more intuitive to have these "helper classes" apply directly to the elements they affect, whenever possible